core/src/main/java/com/linecorp/armeria/internal/client/DecodedHttpResponse.java

```diff
@@ -25,22 +25,22 @@
 import io.netty.channel.EventLoop;
 import io.netty.util.concurrent.EventExecutor;
 
-final class DecodedHttpResponse extends DefaultHttpResponse {
+public final class DecodedHttpResponse extends DefaultHttpResponse {
```

Comment:
I guess a lot of these are residuals before the merge from @ikhoon 's PR.  Can we revert making these classes public if possible?

```suggestion
final class DecodedHttpResponse extends DefaultHttpResponse {
```

Reply:
I moved this class's package `com.linecorp.armeria.client` to `com.linecorp.armeria.internal.client`.
So I need to change access level to `public`.

Reply:
I see 😅   I might've missed this, but was there a reason to move this to the internal package?

Reply:
It was because `DefaultClientRequestContext.initiateConnectionShutdown()` needs to access `HttpSession` but `DefaultClientRequestContext` is in a different package.

Reply:
I see 😅    Sorry about the incorrect comment then.  Please ignore 🙏